src/GuzzleFactory.php

```diff
@@ -81,14 +87,19 @@ public static function make(array $options = [], int $backoff = null, array $cod
      * @param int|null                      $backoff
      * @param int[]|null                    $codes
      * @param \GuzzleHttp\HandlerStack|null $stack
+     * @param int|null                      $retries
      *
      * @return \GuzzleHttp\HandlerStack
      */
-    public static function handler(int $backoff = null, array $codes = null, HandlerStack $stack = null)
+    public static function handler(int $backoff = null, array $codes = null, HandlerStack $stack = null, int $retries = null)
```

Comment:
I'd prefer the parameter order to have stack at the end. We can just make this a major version bump.